examples/streamed-list-objects/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,7 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.6-bin.zip
```

Comment:
Is it correct to escape the colon ":" but not the "\" backslashes?

Reply:
Not sure what you mean
There are only colons here which need to be escaped and forward slashes don't need to be escaped

Reply:
I meant that the ":" colon is escaped, but not the "\\" back slashes. It is more of an observation if that is okay.